python/cm/models.py

```diff
@@ -223,6 +243,45 @@ class ConfigLog(ADCMModel):
 
     __error_code__ = 'CONFIG_NOT_FOUND'
 
+    @transaction.atomic()
+    def save(self, *args, **kwargs):
```

Comment:
It's hard to understand business logic behind this code, I believe unit tests would be helpful here in many ways

Reply:
Added unit tests